src/components/ItaliaTheme/Blocks/Common/SearchFilters/DateRangeFilter/DateRangeFilter.tsx

```diff
@@ -0,0 +1,174 @@
+import { useDateRangePicker } from 'react-aria';
```

Comment:
da quelloo che capisco, correggimi se sbaglio, questo componente DateRangeFilter è generico, non è specifico solamente per i filtri dei blocchi di ricerca, giusto?
Se è così, lo sposterei da /Blocks/Common/SearchFilters
e lo metterei a qualche livello superiore, da usare come componente generico

Reply:
Non e' stato testato al di fuori dello scope del ticket, per questo l'avevo lasciato in questo contesto, ma possiamo spostarlo